setup.py

```diff
@@ -86,6 +86,8 @@ def build_nt2(self):
             print('nt2 archive needed, downloading it')
             url = 'https://github.com/pbrunet/nt2/archive/gemv_release.zip'
             location = urlopen(url)
+            http_code_prefix = location.getcode() / 100
+            assert http_code_prefix not in [4, 5], "Failed to download nt2."
```

Comment:
> I chose an assert because there was one a few lines after :p
> (and because it's short)
> But I'm all ears

This was a real question :-) I'm OK with an assert. I'll accept this on
once Travis is OK